pages/deploy/deploy_aws.md

```diff
@@ -315,6 +329,85 @@ deploy to 2 deployments accounts.
   }
 }
 ```
+
+**Example 3** - Deployment with Custom Authentication 
+
+```json
+{
+  "app": "python ./deploy/app.py",
+  "context": {
+    "@aws-cdk/aws-apigateway:usagePlanKeyOrderInsensitiveId": false,
+    "@aws-cdk/aws-cloudfront:defaultSecurityPolicyTLSv1.2_2021": true,
+    "@aws-cdk/aws-rds:lowercaseDbIdentifier": false,
+    "@aws-cdk/core:stackRelativeExports": false,
+    "tooling_vpc_id": "vpc-1234567890EXAMPLE",
+    "tooling_region": "eu-west-2",
+    "tooling_vpc_restricted_nacl": true,
+    "git_branch": "master",
+    "git_release": true,
+    "quality_gate": false,
+    "resource_prefix": "da",
+    "DeploymentEnvironments": [
+        {
+            "envname": "dev",
+            "account": "000000000000",
+            "region": "eu-west-1",
+            "with_approval": false,
+            "internet_facing": true,
+            "prod_sizing": false,
+             "custom_auth": {
+               "provider": "okta",
+               "url": "https://URL",
+               "redirect_url": "https://redirect_url.net",
+               "client_id": "091sdz02308042",
+               "response_types": "code",
+               "scopes": "openid profile",
+               "jwks_url": "https://jwksurl",
+               "claims_mapping": {
+                 "user_id": "id",
+                 "email": "user_email"
+               }
+             }
+        }
+    ]
+  }
+}
+```
+
+### 6.1 (Important) - Configure data.all to use external Idp
+
+Please read this section only if you are deploying data.all with an external Idp.
+
+Out of the box, data.all uses Cognito as the Idp and the user pool provider. Data.all can also be configured to use an external Idp
```

Comment:
I would first mention that Cognito can be used as a proxy and can be integrated with other IdPs. But for the case where a customer wants to fully replace Cognito then it is possible thanks to "custom_auth" configuration

Reply:
Addressed this. Please take a look